config/releases.exs

```diff
@@ -36,6 +36,7 @@ config :nudge, NudgeWeb.Endpoint,
 # to start each relevant endpoint:
 #
 config :nudge, NudgeWeb.Endpoint, server: true
-#
+  http: [port: {:system, "PORT"}],
+  url: [host: System.get_env("all-friendly-flatfish") <> ".gigalixirapp.com", port: 443]
```

Comment:
please don't hard code this. use `HOSTNAME` env var instead

Reply:
@Menubatenny this line is already there in the same file on L26?
either use `url` or `host` and `port` in that line, and remove this one.